karma.conf.js

```diff
@@ -39,16 +52,18 @@ module.exports = function (config) {
         concurrency: Infinity,
         rollupPreprocessor: {
             plugins: [
-                require('@rollup/plugin-replace')({
-                    '__PACKAGE_NAME__': JSON.stringify(process.env.npm_package_name),
-                    '__PACKAGE_VERSION__': JSON.stringify(process.env.npm_package_version)
+                replace({
+                    '__PACKAGE_NAME__': JSON.stringify(npm_package_name),
+                    '__PACKAGE_VERSION__': JSON.stringify(npm_package_version),
+                    '__HOST_APP_DEV_ORIGIN__': JSON.stringify(devOrigin),
```

Comment:
Any issues with this getting replaced as `undefined`?  Seems like it will get replaced inline and, at most, it would be a warning that the expression is constant.

`if (env === 'localhost' && __HOST_APP_DEV_ORIGIN__) {`

Reply:
Shouldn't be any issues, it will evaluate to `""`, which is falsy and we'll just bypass the block and carry on without checking for a custom localhost origin.